package-lock.json

```diff
@@ -0,0 +1,14255 @@
+{
+  "name": "@chatwoot/chatwoot",
```

Comment:
I believe this file was created by npm, we use yarn already have the yarn.lock. so we won't need this file I guess. 

cc: @pranavrajs

Reply:
I didn't see this. package-lock should be removed, we use yarn

Reply:
@pranavrajs @sojan-official I have removed it in the new commit!